openmetadata-service/src/main/java/org/openmetadata/service/alerts/AlertsRuleEvaluator.java

```diff
@@ -52,7 +52,7 @@ public AlertsRuleEvaluator(ChangeEvent event) {
       examples = {"matchAnySource('bot', 'user')"},
       paramInputType = READ_FROM_PARAM_CONTEXT)
   public boolean matchAnySource(String... originEntity) {
-    if (changeEvent == null) {
+    if (changeEvent == null || changeEvent.getEntityType() == null) {
       return false;
```

Comment:
Please change originEntity to originEntities.

Reply:
updated